share/doc/wake/logging.md

```diff
@@ -4,6 +4,13 @@ This article gives a high level overview of how wake logging works.
 Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
 for commented code.
 
+## Things that Might Appear on Output Streams
+
+* the stdout and stderr of executed Jobs (default to logInfo and logError, respectively, control with setPlanStdout, setPlanStderr)
+* The command line ('echo') of executed Jobs (default to logEcho, control with setPlanEcho)
+* The results of calling print/println (default to logInfo, control with printLevel, printlnLevel)
+* The pass/fail results of ____. (default to ...?)
```

Comment:
This is outside the logger handling.  It's the output of whatever Wake function or expression is called, which is *normally* a pass/fail but does not have to be:
```
$ wake -x '1 + 2'
3
```
This gets printed to Wake's stdout unless `--quiet` is set, but can't be affected by any of the log-level-based flags.